R/tune_race_anova.R

```diff
@@ -244,12 +244,17 @@ tune_race_anova_workflow <-
       )
 
     param_names <- tune::.get_tune_parameter_names(res)
+
     metrics <- tune::.get_tune_metrics(res)
+    metrics <- tune::check_metrics_arg(metrics, object, call = call)
+    opt_metric <- tune::first_metric(metrics)
+    metrics_name <- opt_metric$metric
+    maximize <- opt_metric$direction == "maximize"
 
-    racing_obj_log(res, metrics, control, eval_time)
+    eval_time <- tune::check_eval_time_arg(eval_time, metrics, call = call)
+    metrics_time <- tune::first_eval_time(metrics, metrics_name, eval_time)
```

Comment:
>  it's the time belonging to `opt_metric`

Yes. It is the time that racing is used to choose the best model for the selected metric. 

I can change it to `eval_time_target`. `metrics_time` is the name from a previous PR (tidymodels/tune#782)

Reply:
I'll change it to `opt_metric_time` to be consistent with the other name change (for metric). I'll also make this change in tune since the code is the same.

Reply:
that sounds good 👍